packages/keycloak-service/src/service/serviceClass.ts

```diff
@@ -244,9 +258,10 @@ export class KeycloakAPIService {
   /** delete method
    * Send a DELETE request to the general endpoint
    * Successful requests will result in a HTTP status 204
+   *
    * @param {params} params - the url params object
    * @param {HTTPMethod} method - the HTTP method
-   * @returns the object returned by API
+   * @returns {any} the object returned by API
```

Comment:
Seems that the return is `Promise<any>` instead of just `any`, no?

Reply:
updated